### PR TITLE
Refactor: remove FakerPHP from production 

### PR DESCRIPTION
**Pull request by alaca**
Resolves: [GIVE-1357]

## Description

This PR moves the FakerPHP library from production. 

## Affects

TestData
Model Factories

## Pre-review Checklist

-   [x] Relevant `@unreleased` tags included in DocBlocks
-   [x] [Self Review](https://give.gitbook.io/development-manual/devops/github/code-reviews#self-review) of code and UX completed

[GIVE-1357]: https://stellarwp.atlassian.net/browse/GIVE-1357?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/TestData/ServiceProvider.php [hunk 1]
```diff
@@ -88,4 +97,18 @@ private function addCommands()
...
+     *
+     * @see https://getcomposer.org/doc/07-runtime.md#installed-versions
+     *
+     * @unreleased
+     *
+     * @return bool
+     */
+    private function isFakerInstalled(): bool
+    {
+        return InstalledVersions::isInstalled('fakerphp/faker');
```
Comment (jonwaldstein) [inline src/TestData/ServiceProvider.php:112]: Whoah that's cool 😎

File: composer.json [hunk 2]
```diff
@@ -21,6 +20,7 @@
         "ext-json": "*"
     },
     "require-dev": {
+        "fakerphp/faker": "^1.9",
```
Comment (jonwaldstein) [inline composer.json:23]: I noticed p2p is using faker for something, can you double check that it's not relying on core in production for any reason?

https://github.com/impress-org/give-peer-to-peer/blob/cb9383834c234c597544e40ac3af83b1e2ee9780/src/P2P/Factories/TeamFactory.php#L5

**Review by jonwaldstein — APPROVED** [review]

File: composer.json (hunk 2 above)
Comment (alaca) [reply composer.json:23]: Good catch! I'll check.

**Review by rickalday — APPROVED** [review]
Passed manual and automated QA tests.

**Pull request merged**